### ISSUE TITLE
Improve PUG syntax

### ISSUE THREAD

**Issue by miguelmich** [cuartas15/proyecto#8]
# What
We can improve the PUG syntax a little bit more by making use of the shorthand for classes and default elements, for example in this case: 
https://github.com/cuartas15/proyecto/blob/89ab4f6953d182f83cedf7ee60e09faea0f99197/src/html/index.pug#L2-L4

we can replace it by 
```pug 
.panel-notificacion
  .punto-notificacion
  .espacio-imagen
```

In PUG the element by default is the div (for obvious reasons this is by far the most common HTML tag) so we can ommit it and we can take advantage of the CSS name style notation to create classes attributes or even **id** attrs.

**Comment by cuartas15** [comment]
I was wondering about this, would I use this syntax to save code or use div to improve readability? (specially for me since I'm a noob on this framework and since you didn't specify that in the requirements for the proyect) I chose the latter.

Is not a big of a deal, so that's a sure change.

Thanks

**Comment by miguelmich** [comment]
Fixed on 5c4fe26
